### PR TITLE
#US4R-280: Implemented the possibility to set TX/RX subsequence. 

### PR DESCRIPTION
**Pull request by pjarosik**

### CODE REVIEW

File: api/python/arrus/devices/us4r.py [hunk 1]
```diff
@@ -304,20 +307,30 @@ def get_dto(self):
...
+        )
+
+    def get_data_description_updated_for_subsequence(self, upload_result, sequence):
+        fcm = self._get_fcm(upload_result, sequence)
+        return arrus.metadata.EchoDataDescription(
+            sampling_frequency=self.current_sampling_frequency,
+            custom={"frame_channel_mapping": fcm}
+        )
+
+    def _get_fcm(self, upload_result, sequence):
```
Comment (zkLog) [inline api/python/arrus/devices/us4r.py:323]: Should be some description/docstring for this method.

File: api/python/arrus/ops/us4r.py [hunk 2]
```diff
@@ -215,6 +216,15 @@ def get_sample_range_unique(self):
             raise ValueError("All TX/RXs should acquire the same sample range.")
         return next(iter(sample_range))
 
+    def get_subsequence(self, start, end):
```
Comment (zkLog) [inline api/python/arrus/ops/us4r.py:219]: The same named function with same dosctring is defined in api/python/arrus/ops/imaging.py. Is it intencional?

File: conanfile.txt [hunk 3]
```diff
@@ -5,6 +5,7 @@ gtest/1.10.0
 fmt/7.0.1
 ms-gsl/2.1.0
 eigen/3.3.7
+
```
Comment (zkLog) [inline conanfile.txt:8]: Why leave one empty line?

**Review by zkLog — APPROVED** [review]

File: api/python/arrus/devices/us4r.py [hunk 4]
```diff
@@ -67,6 +67,9 @@ class Us4R(Device, Ultrasound):
     Wraps an access to arrus.core.Us4R object.
     """
 
+    SEQUENCE_START_VAR = "/Us4R:0/sequence:0/start"
```
[the remark this answers is not in the document]
Comment (pjarosik) [reply api/python/arrus/devices/us4r.py:70]: @zkLog currently we only support a single Us4R:0 device connected to the system, therefore Us4R:0 is the only option -- but I will put your remark for the future task...

File: conanfile.txt (hunk 3 above)
Comment (pjarosik) [reply conanfile.txt]: Fixed.

File: api/python/arrus/ops/us4r.py (hunk 2 above)
Comment (pjarosik) [reply api/python/arrus/ops/us4r.py:219]: Yes, that's intentional, be cause we want to be able to both limit the SimpleTxRxSequence and the TxRxSequence to sub-sequence.

File: api/python/arrus/devices/us4r.py (hunk 1 above)
Comment (pjarosik) [reply api/python/arrus/devices/us4r.py:323]: Done.

**Pull request merged**